python/cling/CMakeLists.txt

```diff
@@ -14,13 +14,15 @@ if(ROOT_pyroot_FOUND)
   # TPython::LoadMacro and TPython::Import are broken in the new Cppyy
   # https://bitbucket.org/wlav/cppyy/issues/65
   # For now, we rely on our own implementation of TPython
-  if(NOT MSVC OR "${CMAKE_GENERATOR_PLATFORM}" MATCHES "Win32" OR win_broken_tests)
-    ROOTTEST_ADD_TEST(class MACRO
-                      runPyClassTest.C
-                      WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR}
-                      OUTREF PyClassTest.ref
-                      ENVIRONMENT CLING_STANDARD_PCH=none
-                      CPPYY_BACKEND_LIBRARY=${CMAKE_BINARY_DIR}/lib/libcppyy_backend${PYTHON_UNDER_VERSION_STRING_Development_Main}.so)
+  if(NOT MSVC OR ${LLVM_VERSION} VERSION_LESS 13.0.0 OR llvm13_broken_tests)
```

Comment:
Is `LLVM_VERSION` defined in standalone build of roottest?

Reply:
I hope so, I'll cross-check later

Reply:
Good point, `LLVM_VERSION` is not defined. @vgvassilev is there a way of knowing the LLVM version used by ROOT?